### PR TITLE
Added hyperlink to docker task

### PR DESCRIPTION
**Pull request by DanielSSilva**
Just a small improvement to reference the docker task documentation

### CODE REVIEW

**Comment by PRMerger13** [comment]
@DanielSSilva : Thanks for your contribution! The author(s) have been notified to review your proposed change. @azooinmyluggage

**Comment by ktoliver** [comment]
@juliakm Could you review the proposed update? I think the metadata needs to be changed, IIRC. Thanks!

File: docs/pipelines/ecosystems/containers/acr-template.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ You can also use the Azure portal web UI to create your Azure Container Registry
 
 ## How we build your pipeline
 
-The pipeline that we just created in the previous section was generated from the _Docker container template_ YAML. The build stage uses the Docker task `Docker@2` to build and push your Docker image to the container registry.
+The pipeline that we just created in the previous section was generated from the _Docker container template_ YAML. The build stage uses the [Docker task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/docker?view=azure-devops) `Docker@2` to build and push your Docker image to the container registry.
```
Comment (juliakm) [inline docs/pipelines/ecosystems/containers/acr-template.md:93]:
```suggestion
The pipeline that we just created in the previous section was generated from the _Docker container template_ YAML. The build stage uses the [Docker task](https://docs.microsoft.com/azure/devops/pipelines/tasks/build/docker?view=azure-devops) `Docker@2` to build and push your Docker image to the container registry.
```

**Review by juliakm — APPROVED** [review]

**Comment by PRMerger18** [comment]
@juliakm : Thanks for your contribution! The author(s) have been notified to review your proposed change. @azooinmyluggage

**Comment by juliakm** [comment]
#sign-off

Thanks @DanielSSilva! I made a small change to remove localization from the url string (en-us).

**Comment by PRMerger16** [comment]
@juliakm: I'm sorry - only the author of this article, @azooinmyluggage, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by juliakm** [comment]
@ktoliver Can you merge this change? Looks like we still need to fix the metadata.

**Comment by PRMerger9** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change. @azooinmyluggage

**Pull request merged**